src/freenas/usr/bin/unlock-root-fs.py

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/python3
+
+import json
+import sys
+from subprocess import run
+
+
+def unlock_dataset(entry):
+    if 'RO' not in entry['fhs_entry']['options']:
+        return
+
+    # There shouldn't be a legitimate reason to edit files in /conf
+    if entry['fhs_entry']['name'] == 'conf':
+        return
+
+    print(f'Setting readonly=off on dataset {entry["ds"]}')
+    run(['zfs', 'set', 'readonly=off', entry['ds']])
+
+
+if __name__ == '__main__':
+    datasets = None
+    try:
+        with open('/conf/truenas_root_ds.conf', 'r') as f:
```

Comment:
What is this file? How is it created?

Reply:
It is created during TrueNAS install process c.f. scale-build PR.